lesson-03-assignment

```diff
@@ -0,0 +1,32 @@
+Q1
+topics table (Model association table)
+lesson_topics table (Join table)
+
+Q2
+Topics table
+	id (auto generated primary key)
+	title (the ask for tags) = string → example (“CSS” or “SQL”)
+LessonTopics table
+	lesson_id (foreign key)
+	topic_id (foregin key)
+
+Q3
+I think thats it, maybe description? 
+
+Q4
+topics
+	id (auto generated primary key)
+	title (string)
+topics_lessons
+	id (not required??? but more work not to....)  
```

Comment:
It's pretty rare for disabling the generation of primary keys to be the better option. But you're right in that they might not be strictly necessary. The potential downfalls tend to outweigh the risks though.